ipu-plugin/pkg/ipuplugin/lifecycleservice.go

```diff
@@ -333,6 +335,27 @@ func configureChannel(mode, daemonHostIp, daemonIpuIp string) error {
 	return nil
 }
 
+// sets random bytes for last 2 bytes(5th and 6th) in MAC address
+func setBaseMacAddr() (string, error) {
+	var macAddress string
+	macBytes := make([]byte, 2)
+	_, err := rand.Read(macBytes)
+	if err != nil {
+		return "", fmt.Errorf("error->%v, failed to create random bytes for MAC: ", err)
+	}
+	//Restricting range of last byte in node-policy to be less than 247, to
+	//allow for last byte of MAC+4 for ACC, and last byte of MAC+8 for IMC, as observed in cli-client output.
```

Comment:
Last byte is derived as base_mac_addr[5] = policy mac_addr[5] + func_id

ACC: policy_mac_addr[5] + 4
IMC: policy mac_addr[5] + 5